app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cleared_command: list[str] = command.split()
+    if len(cleared_command) != 3 or cleared_command[0] != "mv":
+        raise (ValueError
+               ("Invalid command format. Use 'mv source destination'."))
```

Comment:
The `raise` statement should not have parentheses around the exception. It should be `raise ValueError("Invalid command format. Use 'mv source destination'.")`.